### PR TITLE
Update Token Binding Reference

### PR DESCRIPTION
**Pull request by chaals**
Addresses the part of #495 that isn't a request for a normative protocol requirement.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/497.html" title="Last updated on Mar 27, 2019, 11:50 PM UTC (a7f451b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/497/1ad7698...a7f451b.html" title="Last updated on Mar 27, 2019, 11:50 PM UTC (a7f451b)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4235,14 +4235,10 @@ <h3>Token Binding</h3>
...
-man-in-the-middle attack</a>. Any protocol using the Verifiable Credentials
-Data Model that requires protection against these kinds of attacks needs to
-perform some sort of token binding, such as
-<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol-18">
-The Token Binding Protocol v1.0</a>, which ties the request for a
-<a>verifiable presentation</a> with the response. Any protocol that does not
-perform token binding is susceptible to man-in-the-middle attacks.
+<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">man-in-the-middle attack</a>.
+Approaches such as token binding [[RFC8471]], which ties the request for a 
+<a>verifiable presentation</a> to the response can secure the protocol.
```
Comment (dlongley) [inline index.html]:
```suggestion
<a>verifiable presentation</a> to the response, can secure the protocol. 
```

File: index.html [hunk 2]
```diff
@@ -4235,14 +4235,10 @@ <h3>Token Binding</h3>
...
-<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">
-man-in-the-middle attack</a>. Any protocol using the Verifiable Credentials
-Data Model that requires protection against these kinds of attacks needs to
-perform some sort of token binding, such as
-<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol-18">
-The Token Binding Protocol v1.0</a>, which ties the request for a
-<a>verifiable presentation</a> with the response. Any protocol that does not
-perform token binding is susceptible to man-in-the-middle attacks.
+<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">man-in-the-middle attack</a>.
+Approaches such as token binding [[RFC8471]], which ties the request for a
```
Comment (msporny) [inline index.html]: Should this be [[?RFC8471]], as this is a non-normative section and thus this should be an informative reference?

**Comment by stonematt** [comment]
WG resolution: https://www.w3.org/2019/04/02-vcwg-minutes.html#resolution03
RESOLUTION: Merge non-normative PR #497 in order to update to an informative reference to RFC8471 raised in issue #495. Close issue #495 once the PR is merged.

**Comment by msporny** [comment]
I'll clean up the `[[?RFC8471]]` reference.

**Pull request merged**